spec/factories/state_file_nj_intakes.rb

```diff
@@ -122,10 +122,6 @@
     routing_number { "011234567" }
     account_number { "123456789" }
     account_type { 1 }
-    primary_signature_pin { '12345' }
```

Comment:
I pulled down this PR, added this here
```
    primary_esigned_at { DateTime.new(2024, 12, 19, 12)}
    spouse_esigned_at { DateTime.new(2024, 12, 18, 12) }
    primary_esigned { 'yes' }
    spouse_esigned { 'yes' }
```
and deleted the attribute assignments in pdf spec. The tests pass. I think maybe these attributes were doing nothing because you weren't assigning primary_esigned and spouse_esigned to yes in this file.

Reply:
Hm, I did try that. 

Agree that it's preferable for these to live in the factory, since all submitted intakes will need to be signed to be submitted. Will give this another shot, thanks for the comment!